.github/workflows/deploy-preview.yml

```
@@ -0,0 +1,50 @@
+name: Deploy Preview
+
+on:
+  push:
+    branches-ignore:
+      - master
```

Should this be `main`?

```suggestion
      - main
```